src/pages/Admin/Venue/DetailsForm.tsx

```diff
@@ -129,13 +152,64 @@ export const DetailsForm: React.FC<DetailsFormProps> = ({
   }, [state]);
 
   const onSubmit = useCallback(
-    async (vals: Partial<FormValues>) => {
-      if (!user) return;
+    async (
+      vals: Partial<FormValues>,
+      userStatuses: UserStatus[],
+      showUserStatuses: boolean
+    ) => {
+      if (!user || formError) return;
       try {
         // unfortunately the typing is off for react-hook-forms.
-        if (!!venueId)
-          await updateVenue({ ...(vals as VenueInput), id: venueId }, user);
-        else await createVenue(vals as VenueInput, user);
+        if (!!venueId) {
+          await updateVenue(
+            {
+              ...(vals as VenueInput),
+              id: venueId,
+              userStatuses,
+              showUserStatus: showUserStatuses,
+            },
+            user
+          );
+
+          console.log(sovereignVenueId);
+
+          if (sovereignVenueId)
+            await updateVenue(
+              {
+                ...(vals as VenueInput),
+                id: sovereignVenueId,
+                parentId: sovereignVenue?.parentId,
+                userStatuses,
+                showUserStatus: showUserStatuses,
+              },
+              user
+            ).then(() => {
+              console.log(sovereignVenue);
+
+              if (sovereignVenue) {
+                console.log(
+                  "updating redux sov venue",
+                  showUserStatuses,
+                  userStatuses
+                );
+                dispatch(
+                  setSovereignVenue({
+                    ...sovereignVenue,
+                    userStatuses,
+                    showUserStatus: showUserStatuses,
+                  })
+                );
```

Comment:
are you here changing the sovereing venue instead of the one that is edited right now? not sure if it's the desired behavior. I think it can cause bugs. I think we should let the admin decide what venue to edit, not touching others.

Reply:
This is somewhat safe because it updates only the sov venue which is actually static after update because it's not using the live hook and this manual update is necessary here.

Reply:
The venue itself is using live updates because of the use connect hook.

Reply:
i didn't mean that we might update a wrong venue (I trust sovereignVenue hook).
I meant that clicking Save button while editing venue I do not expect that this "saving" edits other venues as well. It's weird for me...

Reply:
This is the expected flow, after changing a venue the whole 'world' should use these statuses and that's why i'm using the sovereign venue as a source

Reply:
@sunny-viktoryia there was some discussion about this on slack in https://doesitsparkle.slack.com/archives/C01H9D1P1PE/p1625734647006000

If these fields are stored on the sovereign venue, then that's kind of where we need to be able to add them. I'd say this is more a hangover of the legacy'ness of sovereign venue, compared to how it should be in world architecture where these settings will sit at the 'world' level.

It might be good UX to include some info in the UI near these controls though to say that they are stored on the sovereign venue (and to show the sovereign venue's name and/or ID there as well). That would seem like a good middleground to me.